Quaver.Shared/Screens/Result/UI/ResultScoreContainer.cs

```diff
@@ -270,6 +280,125 @@ private void CreateKeyValueItems()
             };
         }
 
+        /// <summary>
+        ///     Creates all of the statistics items.
+        /// </summary>
+        private void CreateStatisticsKeyValueItems()
+        {
+            var processor = Screen.GetScoreProcessor();
+            var largestHitWindow = processor.JudgementWindow.Values.Max();
+
+            double mean = 0.0, standardDeviation = 0.0;
+            if (processor.Stats != null)
+            {
+                var hitDifferences = new List<int>();
+                var sum = 0;
+
+                foreach (var breakdown in processor.Stats)
+                {
+                    var hitDifference = breakdown.HitDifference;
+
+                    // The LN releases are _not_ scaled here because we want an accurate mean.
+
+                    // No need to check for Type == Miss as all of them have hitDifference == int.MinValue.
+                    if (hitDifference != int.MinValue && Math.Abs(hitDifference) <= largestHitWindow)
+                    {
+                        hitDifferences.Add(hitDifference);
+                        sum += hitDifference; // This overflows at like 13 million max judgements.
+                    }
+                }
+
+                var count = hitDifferences.Count();
+                if (count > 0)
+                {
+                    mean = (double) sum / (double) count;
+                    standardDeviation = Math.Sqrt(hitDifferences.Average(v => Math.Pow(v - mean, 2)));
+
+                    // Undo the rate scaling.
+                    mean /= ModHelper.GetRateFromMods(processor.Mods);
+                    // Since variance(ax) = a^2 variance(x), then std(ax) = a std(x)
+                    standardDeviation /= ModHelper.GetRateFromMods(processor.Mods);
+                }
+            }
+
+            var availableWidth = Width - VerticalDividerLine.X;
+            var buttonPadding = 15;
+            var buttonWidth = (availableWidth - buttonPadding * 3) / 2;
+
+            var localOffsetButton = new BorderedTextButton("Adjust Local Offset", Colors.MainAccent,
+                (o, e) =>
+                {
+                    // Local offset is scaled with rate, so the adjustment depends on the rate the
+                    // score was played on.
+                    var change = mean * ModHelper.GetRateFromMods(processor.Mods);
+                    ResultScreen.Map.LocalOffset =
+                        (int) Math.Round(ResultScreen.Map.LocalOffset - change);
```

Comment:
Put on the same line

Reply:
If `change` isn't being used anywhere else, there's no need to make a local variable for it

Reply:
The local variable makes the expression more readable.